rust/kcl-lib/src/std/args.rs

```diff
@@ -487,6 +487,26 @@ impl<'a> FromKclValue<'a> for Vec<KclValue> {
     }
 }
 
+impl<'a> FromKclValue<'a> for Vec<SketchFaceOrTaggedFace> {
+    fn from_kcl_val(arg: &'a KclValue) -> Option<Self> {
+        let items = arg
+            .clone()
+            .into_array()
+            .iter()
+            .map(|v| {
```

Comment:
Could this whole `map` just be `            .map(|v| SketchFaceOrTaggedFace::from_kcl_val(v))`